examples/api/lib/widgets/safe_area/safe_area.0.dart

```diff
@@ -122,6 +122,7 @@ enum Inset implements Value {
     Text(label),
     Builder(
       builder: (BuildContext context) => Slider(
+        key: ValueKey<Inset>(this),
```

Comment:
why is this needed?

Reply:
I did it to make the testing easier… now that you mention it, I could probably just loop through every slider that the finder returns.

Reply:
Factored this out in https://github.com/flutter/flutter/pull/159344/commits/dfef025ba68c7daff5c0eb3e81855dac8f6fa419!